server/rpm/pbench-server.spec.j2

```diff
@@ -94,6 +94,9 @@ fi
 /%{installdir}/%{static}/VERSION
 /%{installdir}/%{static}/package.json
 /%{installdir}/package.json
+/%{installdir}/setup.py
```

Comment:
Not sure why we need `setup.py` and `setup.cfg`.

Reply:
I'm just confused - fixed.